js/adapt-contrib-matching.js

```diff
@@ -278,6 +278,67 @@ define([
             this.$('select').eq(i).val(value).trigger('change');
         },
 
+        /**
+        * Used by tracking extensions to return an object containing the component's specific interactions.
+        */
+        getInteractionObject: function() {
+            var interactions = {
+                correctResponsesPattern: null,
+                source: null,
+                target: null 
+            };
```

Comment:
Is it necessary to define the properties as `null` if you're setting them directly below?